plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestSqlServerIntegrationSmokeTest.java

```diff
@@ -415,6 +415,8 @@ public void testShowCreateForPartitionedTablesWithDataCompression()
                         "   salesdate date,\n" +
                         "   quantity integer\n" +
                         ")");
+
+        assertUpdate("DROP TABLE partitionedSales");
```

Comment:
don't you need to drop `pfSales` and `psSales`? The best would be to use `io.trino.testing.sql.TestTable` (modified somehow) for things like that.